### PR TITLE
vim-patch:8.1.1824

### PR DESCRIPTION
**Pull request by janlazo**

### CODE REVIEW

File: src/nvim/testdir/test_spell.vim [hunk 1]
```diff
@@ -303,7 +304,7 @@ func Test_zz_sal_and_addition()
   mkspell! Xtest Xtest
   set spl=Xtest.latin1.spl spell
   call assert_equal('kbltykk', soundfold('goobledygoook'))
-  call assert_equal('kprnfn', soundfold('k�op�r�n�ven'))
+  call assert_equal('kprnfn', soundfold('kóopërÿnôven'))
```
Comment (janlazo) [inline src/nvim/testdir/test_spell.vim]: Not part of the patch but I saved the file with `set encoding=utf-8`.

Comment (blueyed) [reply src/nvim/testdir/test_spell.vim]: Then the added note is not true anymore however - better to keep it like Vim then I guess.

Comment (janlazo) [reply src/nvim/testdir/test_spell.vim]: Redid it with latin1 encoding in Vim.
